src/Sulu/Bundle/PreviewBundle/SuluPreviewBundle.php

```diff
@@ -29,7 +29,7 @@ public function build(ContainerBuilder $container)
 
         $container->addCompilerPass(
             new TaggedServiceCollectorCompilerPass(
-                'sulu_preview.preview',
```

Comment:
@wachterjohannes Is this a BC break? What if someone has overwritten that service?

Reply:
no this is not a BC break - a BC break which should be documented is the constructor change